src/data/zwave_js.ts

```diff
@@ -177,6 +178,15 @@ export interface RequestedGrant {
 
 export const nodeStatus = ["unknown", "asleep", "awake", "dead", "alive"];
 
+export const securityClass = {
```

Comment:
We already have this here: <[`69b8179` (#10132)](https://github.com/home-assistant/frontend/pull/10132/commits/69b8179ec20ab15a18a595077bf6f42060a10eba#diff-b477815d091e8935bc5d8576f815ee7cccb7373428ea4ffc955a965ce6a0684dR44)>